examples/nfs/README.md

```diff
@@ -1,38 +1,27 @@
 <!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
 
-<!-- BEGIN STRIP_FOR_RELEASE -->
 
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
-<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
-     width="25" height="25">
+<!-- END MUNGE: UNVERSIONED_WARNING -->
 
-<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+# Outline
 
-If you are using a released version of Kubernetes, you should
-refer to the docs that go with that version.
+![alt text][nfs pv example]
 
 <!-- TAG RELEASE_LINK, added by the munger automatically -->
 <strong>
 The latest release of this document can be found
 [here](http://releases.k8s.io/release-1.2/examples/nfs/README.md).
 
-Documentation for other releases can be found at
-[releases.k8s.io](http://releases.k8s.io).
-</strong>
---
+As illustrated above, two persistent volumes are used in this example:
 
-<!-- END STRIP_FOR_RELEASE -->
+- Web frontend Pod uses a persistent volume based on NFS server, and
+- NFS server uses an auto provisioned persistent volume from GCE PD or AWS EBS.
 
-<!-- END MUNGE: UNVERSIONED_WARNING -->
+Note, this is a *not* a production ready NFS setup.
```

Comment:
Why isn't it production ready?  What would need to change to make it production ready?  Is making it production ready even feasible, or are there any production read examples we can link to?